app/models/order_item.rb

```diff
@@ -31,6 +31,7 @@ def set_defaults
 
   def update_message(level, message)
     progress_messages << ProgressMessage.new(:level => level, :message => message, :tenant_id => self.tenant_id)
+    Rails.logger.info("Order: #{order.id}, Request ID: #{insights_request_id}, #{level} message: #{message}")
```

Comment:
It would make more sense to call `logger` with the proper level instead of writing `level` in the log message.   Also would referencing  the `order_id` column avoid loading the `order` object?

Something like this:
```ruby
Rails.logger.send(level, "Order: #{order_id}, Request ID: #{insights_request_id}, message: #{message}")
```

Reply:
Oh yeah, definitely. It looks like we only ever use info/error which are valid error levels. Hacky first look solution can always be improved!